src/domain/delegate/delegate.repository.interface.ts

```diff
@@ -21,7 +21,7 @@ export interface IDelegateRepository {
     delegator?: string,
     signature?: string,
     label?: string,
-  ): Promise<unknown>;
+  ): Promise<void>;
```

Comment:
This endpoint is expected to return no data, just the 200 status code, so I changed the return type of the related functions in the interfaces, in order to make this fact explicit.